contracts/drafts/ERC777/ERC777.sol

```diff
@@ -40,16 +37,17 @@ contract ERC777 is IERC777 {
     mapping(address => mapping(address => bool)) private _operators;
     mapping(address => mapping(address => bool)) private _revokedDefaultOperators;
 
+    // ERC20-allowances
+    mapping (address => mapping (address => uint256)) private _allowances;
+
     constructor(
         string memory name,
         string memory symbol,
         uint256 granularity,
         address[] memory defaultOperators
-    ) public {
-        require(granularity > 0);
+    ) public ERC20Detailed(name, symbol, 18) { // The spec requires that decimals be 18
```

Comment:
If the token is also an ERC20 token, it MUST be registered as "ERC20Token" in ERC1820 registry

Reply:
:scream: can't believe I forgot about this, thank you!